src/test/java/no/sikt/nva/pubchannels/handler/fetch/serialpublication/FetchSerialPublicationByIdentifierAndYearHandlerTest.java

```diff
@@ -0,0 +1,407 @@
+import static no.sikt.nva.pubchannels.handler.TestUtils.constructRequest;
+import static no.sikt.nva.pubchannels.handler.TestUtils.createPublicationChannelUri;
+import static no.sikt.nva.pubchannels.handler.TestUtils.mockChannelRegistryResponse;
+import static no.sikt.nva.pubchannels.handler.TestUtils.mockRedirectedClient;
+import static no.sikt.nva.pubchannels.handler.TestUtils.mockResponseWithHttpStatus;
+import static no.sikt.nva.pubchannels.handler.TestUtils.randomYear;
+import static no.sikt.nva.pubchannels.handler.TestUtils.setupInterruptedClient;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.when;
+import com.google.common.net.MediaType;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpResponse;
+import java.util.Map;
+import java.util.UUID;
+import no.sikt.nva.pubchannels.handler.TestChannel;
+import no.sikt.nva.pubchannels.handler.fetch.FetchByIdentifierAndYearHandlerTest;
+import no.sikt.nva.pubchannels.handler.fetch.series.FetchSeriesByIdentifierAndYearHandler;
+import no.sikt.nva.pubchannels.handler.search.serialpublication.SerialPublicationDto;
+import nva.commons.apigateway.GatewayResponse;
+import nva.commons.core.paths.UriWrapper;
+import nva.commons.logutils.LogUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.zalando.problem.Problem;
+
+public class FetchSerialPublicationByIdentifierAndYearHandlerTest extends FetchByIdentifierAndYearHandlerTest {
+
+    private static final String SERIES_IDENTIFIER_FROM_CACHE = "50561B90-6679-4FCD-BCB0-99E521B18962";
+    private static final String SERIES_YEAR_FROM_CACHE = "2024";
+    private static final URI SELF_URI_BASE = URI.create(
+        "https://localhost/publication-channels/" + SERIAL_PUBLICATION_PATH);
+    private static final String CHANNEL_REGISTRY_PATH_ELEMENT = "/findjournalserie/";
+
+    @BeforeEach
+    void setup() {
+        this.handlerUnderTest = new FetchSerialPublicationByIdentifierAndYearHandler(environment,
+                                                                                     this.channelRegistryClient,
+                                                                                     this.cacheService,
+                                                                                     super.getAppConfigWithCacheEnabled(
+                                                                                         false));
+    }
+
+    @ParameterizedTest
```

Comment:
Can you add fixed/deterministic names to all parameterized tests? For example here, `@ParameterizedTest(name = "Should return correct data for type \"{0}\"")`

If we don't explicitly set a name for parameterized tests, they get a random name on each run. This causes the GitHub action to count them as "new" tests and we get the "200 tests added, 200 tests removed" spam.